bloom-instance/alb.tf

```diff
@@ -0,0 +1,61 @@
+# Anything related to our Application Load Balancer goes here
+
+resource "aws_lb" "public_alb" {
+  name               = "${var.resource_prefix}-${terraform.workspace}-Public-ALB"
+  internal           = false
+  load_balancer_type = "application"
+  security_groups    = [aws_security_group.public_alb.id]
+
+  # Note that an ALB requires at least 2 subnets
+  subnets = [for subnet in aws_subnet.public : subnet.id]
+
+  enable_deletion_protection = false
+
+  access_logs {
+    bucket  = aws_s3_bucket.logging_bucket.id
+    prefix  = "public-alb-logs"
+    enabled = true
+  }
+}
+
+# A Listener defines how to handle certain requests
+# This Listener returns a 404 status code for any request on port 80
+resource "aws_lb_listener" "alb_listener" {
+  load_balancer_arn = aws_lb.public_alb.arn
+  port              = "80"
+  protocol          = "HTTP"
+
+  default_action {
+    type = "fixed-response"
+
+    fixed_response {
+      content_type = "text/plain"
+      status_code  = "404"
+    }
+  }
+}
+
+# ALBs need Security Groups, too
+# This one enables access from any external IP
+resource "aws_security_group" "public_alb" {
+  name_prefix = "${var.resource_prefix}-Public-ALB"
```

Comment:
Thanks for making the previous changes! Now we just need this name to be updated the same way.